Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -1192,31 +1192,31 @@ void WebsiteDataStore::isGrandfathered(const URL& url, CompletionHandler<void(bo
 void WebsiteDataStore::setPrevalentResource(const URL& url, CompletionHandler<void()>&& completionHandler)
 {
     ASSERT(RunLoop::isMain());
-    
+
     if (url.protocolIsAbout() || url.isEmpty()) {
         completionHandler();
         return;
     }
-    
+
```

Comment:
Please revert all the noise from your PR. It makes your PR more painful to review than it needs to be.